app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,27 @@
+class NotificationsController < ApplicationController
+  def index; end
+
+  def create
+    if params[:consent].nil?
+      # TODO: Better Error messages once wired up to API
+      flash[:error] = "Please select one option"
+      redirect_to notifications_path
+      return
+    end
+
+    if params.require(:consent).present?
+      flash[:success] = "Your notification preferences have been saved."
```

Comment:
Is the alert text a candidate for locales?

Reply:
After the discussion with Dan, I don't think we need to worry about the locales for the moment.